homeassistant/components/netgear/config_flow.py

```diff
@@ -135,6 +135,9 @@ async def async_step_ssdp(self, discovery_info: ssdp.SsdpServiceInfo) -> FlowRes
 
         _LOGGER.debug("Netgear ssdp discovery info: %s", discovery_info)
 
+        if ssdp.ATTR_UPNP_SERIAL not in discovery_info.upnp:
```

Comment:
Please add a test for this branch

Reply:
Thanks, forgot that.
Test has now been added.